### PR TITLE
Updating documentation for latest docker images

### PR DESCRIPTION
**Pull request by anki2189**
#1595

### CODE REVIEW

**Comment by anki2189** [comment]
#https://github.com/strongbox/strongbox/issues/1595

File: docs/assets/resources/docker/bashrc-strongbox [hunk 1]
```diff
@@ -90,33 +90,33 @@ dockerLocal() {
 }
 
 dockerMvn35Checkout() {
-    dockerCheckout strongboxci/alpine:jdk8-mvn-3.5 "$@"
+    dockerCheckout strongboxci/alpine:jdk8-mvn3.6 "$@"
 }
```
Comment (steve-todorov) [inline docs/assets/resources/docker/bashrc-strongbox]: Could you also change the function names from `dockerMvn35Checkout` to `dockerMvnCheckout` (and so on)?

Comment (anki2189) [reply docs/assets/resources/docker/bashrc-strongbox]: okay will do.

**Review by steve-todorov — APPROVED** [review]
LGTM! Many thanks for fixing all of these! :)

**Pull request merged**